apps/cli/src/index.ts

```diff
@@ -12,6 +12,8 @@ import { exec as execCallbackBased } from "node:child_process";
 import { create as createTar } from "tar";
 import { request as insecureRequest } from "node:http";
 import { request as secureRequest } from "node:https";
+import projects from "./projects";
+import clusters from "./clusters";
```

Comment:
As confirmed by [these docs](https://code.visualstudio.com/docs/typescript/typescript-refactoring#_organize-imports), that behaviour is because of this line in your file:
```json
      "source.organizeImports": "explicit",
```
I think it's good to sort imports on-save, so I have set up the same setting locally.